### DISCUSSION TITLE
What does the "Home" icon in Perograms: "Action" mean?

### DISCUSSION THREAD

**Discussion by zk637** [tnodir/fort#249] [Q&A] [answered]
Different ICONS appear, which I thought at first was a wildcard match But apparently not, other wildcard rules do not have this icon

![image](https://github.com/tnodir/fort/assets/78423939/41e2fea7-9453-4603-be1c-dd846c134d95)

**Comment by zk637** [comment]
supplementary instruction,

I think I found the action that triggered the scene, copy the full path of the exe file (Everything is used here), manually add the full path using the copy, and the icon will become the one shown in the figure. But I still don't know what it means

![image](https://github.com/tnodir/fort/assets/78423939/8de9ad62-5c6d-4ea7-984c-e07863e9fa16)


![image](https://github.com/tnodir/fort/assets/78423939/dc6fda0c-246b-4351-8f79-a81c02636679)


![image](https://github.com/tnodir/fort/assets/78423939/46905f87-2c18-4076-86ec-097ef5773e1b)

**Reply by tnodir** [reply, marked as answer]
This icon is for "Block Internet Traffic" flag:

![fort-app-block-inet](https://github.com/tnodir/fort/assets/303655/bb2dd6aa-6617-4a10-a553-4b301385812c)

**Reply by zk637** [reply]
Thank you. I guess I got mixed up. Can the mouse hover display the corresponding behavior description? (Example: Allow, Block, Kill Process, Block Internet Traffic) like this:

![image](https://github.com/tnodir/fort/assets/78423939/5a50cdb5-2db8-42ba-9bcd-6d19e011a2c5)


And I also have a small suggestion, can you add an attempt under "Home" to achieve the absolute path and wildcard display respectively

exmaple: Displays in the box together

![image](https://github.com/tnodir/fort/assets/78423939/8b378a9c-10f0-4f14-8f3b-37a63e84e6af)

![image](https://github.com/tnodir/fort/assets/78423939/38873e9b-c361-455e-870b-9463fe437823)

**Reply by tnodir** [reply]
> Can the mouse hover display the corresponding behavior description? (Example: Allow, Block, Kill Process, Block Internet Traffic)

OK, let me improve it.

> And I also have a small suggestion, can you add an attempt under "Home" to achieve the absolute path and wildcard display respectively

Do you mean a grouping by wildcard paths?

**Reply by zk637** [reply]
Yup. Which now displays absolute path and wildcard cross displays, makes it difficult to manage all wildcards intuitively

**Reply by tnodir** [reply]
Do you want it to *always* group by wildcard/absolute paths, not regarding current sorting column?

Or we can add optional "Filter flags" to show only for example "Wildcard paths"?

**Reply by zk637** [reply]
Or we can add optional "Filter flags" to show only for example "Wildcard paths"? Better because it makes us focus with wildcards

**Reply by tnodir** [reply]
OK, I'll implement it later: #250.

**Reply by tnodir** [reply]
@zk637 Implemented in [v3.12.9](https://github.com/tnodir/fort/releases/tag/v3.12.9).

**Reply by zk637** [reply]
Well, it's running as scheduled, thanks for your efforts

![image](https://github.com/tnodir/fort/assets/78423939/051feb98-cb41-42da-8fda-ac99f0e51c92)

**Reply by tnodir** [reply]
There is a bug now: searching by text does not work when the "Wildcard Paths" filter is active.

Fixed by next release.

**Comment by zk637** [comment]
> @zk637 Implemented in [v3.12.9](https://github.com/tnodir/fort/releases/tag/v3.12.9).
